ductus/tools/utils.py

```diff
@@ -566,3 +566,53 @@ def filter_experiment(sample_sheet_file):
         return False
     else:
         return True
+    
+
+def get_no_expected_fastqs(sample_sheet_file, file_list):
```

Comment:
Is get_nr_expected_fastqs a better name? no suggests a check if there is no fastq files.

Reply:
Yes, it is a much better name!